perception/lidar_centerpoint_tvm/lib/centerpoint_tvm.cpp

```diff
@@ -0,0 +1,270 @@
+  const tvm_utility::pipeline::InferenceEngineTVMConfig & config,
+  const CenterPointConfig & config_mod)
+: datatype_bytes(config.tvm_dtype_bits / 8), config_detail(config_mod)
+{
+  head_out_heatmap.resize(
+    config.network_outputs[0].second[1] * config.network_outputs[0].second[2] *
+    config.network_outputs[0].second[3]);
+  head_out_offset.resize(
+    config.network_outputs[1].second[1] * config.network_outputs[1].second[2] *
+    config.network_outputs[1].second[3]);
+  head_out_z.resize(
+    config.network_outputs[2].second[1] * config.network_outputs[2].second[2] *
+    config.network_outputs[2].second[3]);
+  head_out_dim.resize(
+    config.network_outputs[3].second[1] * config.network_outputs[3].second[2] *
+    config.network_outputs[3].second[3]);
+  head_out_rot.resize(
+    config.network_outputs[4].second[1] * config.network_outputs[4].second[2] *
+    config.network_outputs[4].second[3]);
+  head_out_vel.resize(
+    config.network_outputs[5].second[1] * config.network_outputs[5].second[2] *
+    config.network_outputs[5].second[3]);
+}
+
+std::vector<Box3D> BackboneNeckHeadPostProcessor::schedule(const TVMArrayContainerVector & input)
+{
+  // float32_t * heatmap_ptr = static_cast<float32_t *>(input[0].getArray()->data);
+  // float32_t * offset_ptr = static_cast<float32_t *>(input[1].getArray()->data);
+  // float32_t * z_ptr = static_cast<float32_t *>(input[2].getArray()->data);
+  // float32_t * dim_ptr = static_cast<float32_t *>(input[3].getArray()->data);
+  // float32_t * rot_ptr = static_cast<float32_t *>(input[4].getArray()->data);
+  // float32_t * vel_ptr = static_cast<float32_t *>(input[5].getArray()->data);
+
+  // head_out_heatmap.assign(heatmap_ptr, heatmap_ptr + head_out_heatmap.size());
+  // head_out_offset.assign(offset_ptr, offset_ptr + head_out_offset.size());
+  // head_out_z.assign(z_ptr, z_ptr + head_out_z.size());
+  // head_out_dim.assign(dim_ptr, dim_ptr + head_out_dim.size());
+  // head_out_rot.assign(rot_ptr, rot_ptr + head_out_rot.size());
+  // head_out_vel.assign(vel_ptr, vel_ptr + head_out_vel.size());
+
+  TVMArrayCopyToBytes(
+    input[0].getArray(), head_out_heatmap.data(), head_out_heatmap.size() * datatype_bytes);
+  TVMArrayCopyToBytes(
+    input[1].getArray(), head_out_offset.data(), head_out_offset.size() * datatype_bytes);
+  TVMArrayCopyToBytes(input[2].getArray(), head_out_z.data(), head_out_z.size() * datatype_bytes);
+  TVMArrayCopyToBytes(
+    input[3].getArray(), head_out_dim.data(), head_out_dim.size() * datatype_bytes);
+  TVMArrayCopyToBytes(
+    input[4].getArray(), head_out_rot.data(), head_out_rot.size() * datatype_bytes);
+  TVMArrayCopyToBytes(
+    input[5].getArray(), head_out_vel.data(), head_out_vel.size() * datatype_bytes);
+
+  std::vector<Box3D> det_boxes3d;
+
+  generateDetectedBoxes3D(
+    head_out_heatmap, head_out_offset, head_out_z, head_out_dim, head_out_rot, head_out_vel,
+    config_detail, det_boxes3d);
+
+  return det_boxes3d;
```

Comment:
I don't think we need to make this a pointer because in reality the size of this vector is small, generally less than 100 objects. And since `det_boxes3d` is passed from the very top level, several APIs have to be changed as well. I don't think there is too much space for improvement of performance. 
https://github.com/liuzf1988/autoware.universe/blob/7dfd809047b34ee96dde86ef7336f4d1772e9b93/perception/lidar_centerpoint_tvm/src/node.cpp#L119